src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/StateTests.cs

```diff
@@ -410,6 +410,8 @@ private static void AssertEncryptedContentEqual(byte[] expected, byte[] actual)
             Assert.Equal<byte>(expected, actual);
         }
 
+        // As of OpenSSL 1.0.2g, there's no support for adding originator certificates to a newly instantiated CMS_ContentInfo structure
+        // so we can't support this for the encryption process.
         private static bool EncryptionSupportsAddingOriginatorCerts =>
```

Comment:
Rather than having N copies of this, can you put it in one class and forward to the one implementation for the rest?